src/components/itemContextMenu.js

```diff
@@ -294,6 +294,14 @@ export async function getCommands(options) {
         });
     }
 
+    if (canPlay) {
+        commands.push({
+            name: globalize.translate('ShowDetails'),
+            id: 'navigatetodetailpage',
```

Comment:
Maybe call id `itemdetails` like `artist`, `album` and `lyrics`?

Reply:
@dmitrylyzo thanks for your feedback!  
I've realized I forgot to hide the "Details" link when the user is already on the details page. I tried searching the source for similar `appRouter` conditions but couldn't find anything relevant.

Could you point me in the right direction?